packages/webview_flutter/webview_flutter/test/webview_flutter_test.dart

```diff
@@ -1114,6 +1114,58 @@ void main() {
     });
   });
 
+  group('allowsLinkPreview', () {
+    testWidgets('defaults to true', (WidgetTester tester) async {
+      await tester.pumpWidget(const WebView());
+
+      final CreationParams params = captureBuildArgs(
+        mockWebViewPlatform,
+        creationParams: true,
+      ).single as CreationParams;
+
+      expect(params.webSettings!.allowsLinkPreview, true);
+    });
+
+    testWidgets('can be disabled', (WidgetTester tester) async {
+      await tester.pumpWidget(const WebView(
+        allowsLinkPreview: false,
+      ));
+
+      final CreationParams params = captureBuildArgs(
+        mockWebViewPlatform,
+        creationParams: true,
+      ).single as CreationParams;
+
+      expect(params.webSettings!.allowsLinkPreview, false);
+    });
+
+    testWidgets('can be changed', (WidgetTester tester) async {
```

Comment:
This test is failing because the `allowsLinkPreview` values needs to be updated at https://github.com/flutter/plugins/blob/main/packages/webview_flutter/webview_flutter/lib/src/webview.dart#L390.

Reply:
I have removed `allowsLinkPreview` from `WebSettings`, I think that is correct?

Reply:
No, you still want `allowsLinkPreview` in `WebSettings` so that updates can be made to it. I would also re-add the test that were removed for it. You just need to add the updates to `allowsLinkPreview` in the method mentioned above. e.g.

```dart
WebSettings _clearUnchangedWebSettings(
    WebSettings currentValue, WebSettings newValue) {
  assert(currentValue.javascriptMode != null);
  /* other values */
  assert(newValue.allowsLinkPreview != null);

  JavascriptMode? javascriptMode;
  /* other values */
  bool? allowsLinkPreview;
  if (currentValue.javascriptMode != newValue.javascriptMode) {
    javascriptMode = newValue.javascriptMode;
  }
  /* other values */
  if (currentValue.allowsLinkPreview != newValue.allowsLinkPreview) {
    allowsLinkPreview = newValue.allowsLinkPreview;
  }

  return WebSettings(
    javascriptMode: javascriptMode,
    /* other values */
    allowsLinkPreview: allowsLinkPreview,
  );
}
```

Reply:
Sorry for the misunderstanding; that's my bad. I have added it back in and made the changes